src/main/rx/rx.c

```diff
@@ -733,6 +751,14 @@ void updateRSSI(timeUs_t currentTimeUs)
     default:
         break;
     }
+#ifdef USE_RX_LINK_QUALITY_INFO
+    if (linkQualitySource == LQ_SOURCE_RX_CHANNEL) {
+        // Read value of AUX channel
+        int16_t pwmLinkQuality = rcData[rxConfig()->link_quality_channel - 1];
```

Comment:
Is this something that the CRSF hardware can be set up to emit, or why is this in here?

Reply:
its a large change i know .. hard to read  a comment .. should have been done ..
Support multiple RSSI sources (FrSky redundancy) #7139

 { "link_quality_channel",       VAR_INT8   | MASTER_VALUE, .config.minmax = { 0, MAX_SUPPORTED_RC_CHANNEL_COUNT }, PG_RX_CONFIG, offsetof(rxConfig_t, link_quality_channel) }, 
 and that  should 
#ifdef USE_RX_LINK_QUALITY_INFO
    if (rxConfig()->link_quality_channel > 0) {
        rssiSource = RSSI_SOURCE_RX_CHANNEL;
    }
#endif
now i can read the secondary aux for another second  rssi .. 
did i over look something ?
Should rename  LQ_SOURCE_RX_CHANNEL to  LQ_SOURCE_RX_CHANNEL_AS_2ND_RSSI  to make it clear .....

Reply:
Wait what? You are reading something that is RSSI (coming from where?), and feeding it into link quality? This does not sound right.
- please keep your pull requests to one functional change (or at least one set of related changes) each - this makes it easier to discuss them;
- I still don't think that inputs that are (or act like) RSSI and inputs that are (or act like) link quality should be mixed - they work differently, and mixing them will just be confusing.

Reply:
Sorry for the confusion I split out once can split again if you like ?
I don't mean to make it hard for you you have a lot on your plate

Reply:
Ie just crsf and anothe to discuss aux channel for link and second rssi

Reply:
see pr Link Quality multiple sources without Aux DUP of 8232 #8247
will close this or the other .

Reply:
Yes please, pull requests should only contain one feature, or at least related features - the 'link quality channel' seems to be an entirely different idea, and need more discussion.

Reply:
already did  closing this  ref only .
see pr Link Quality multiple sources without Aux DUP of 8232 #8247